sources/packages/api/src/services/application/application.service.ts

```diff
@@ -675,35 +685,133 @@ export class ApplicationService extends RecordDataModelService<Application> {
 
   /**
    * Get all active applications of an institution location
-   * with application_status is completed
+   * with application status completed and respective archive status.
    * @param locationId location id .
+   * @param paginationOptions options to execute the pagination.
+   * @param applicationStatus View status of applications requested by user.
    * @returns Student Active Application list.
    */
-  async getActiveApplications(locationId: number): Promise<Application[]> {
+  async getActiveApplications(
+    locationId: number,
+    paginationOptions: PaginationOptions,
+    archived: boolean,
+  ): Promise<PaginatedResults<Application>> {
     // TODO: there are two similar methods to get one and many records for the same list/details getActiveApplication and getActiveApplications. Can we use only one?
-    return this.repo
+    const activeApplicationQuery = this.repo
       .createQueryBuilder("application")
       .select([
         "application.applicationNumber",
         "application.id",
         "currentAssessment.id",
-        "application.applicationStatus",
         "offering.studyStartDate",
         "offering.studyEndDate",
         "student.id",
         "user.firstName",
         "user.lastName",
+        "application.isArchived",
+        "studentScholasticStanding.id",
       ])
       .leftJoin("application.currentAssessment", "currentAssessment")
       .leftJoin("currentAssessment.offering", "offering")
+      .leftJoin(
+        "currentAssessment.studentScholasticStanding",
+        "studentScholasticStanding",
+      )
       .innerJoin("application.student", "student")
       .innerJoin("student.user", "user")
       .where("application.location.id = :locationId", { locationId })
       .andWhere("application.applicationStatus = :applicationStatus", {
         applicationStatus: ApplicationStatus.completed,
-      })
-      .orderBy("application.applicationNumber", "DESC")
-      .getMany();
+      });
+
+    activeApplicationQuery.andWhere("application.isArchived = :isArchived", {
+      isArchived: archived,
+    });
+
+    if (archived) {
+      activeApplicationQuery.orWhere(
+        new Brackets((qb) => {
+          qb.where("studentScholasticStanding.id IS NOT NULL");
+        }),
+      );
+    }
+
+    activeApplicationQuery.orderBy("application.applicationNumber", "DESC");
```

Comment:
this order by will be re-written by order by at 755. Use addOrderBy at 755. 

Also put all the OrderBy sequentially.